Updated styles section (#76)
strapi/documentation: developer-docs/latest/admin-panel/customization.md

The AdminUI package source can be easily found in `./node_modules/strapi-admin/src/`.

> For example, to change the top-left displayed admin panel's color, copy the `./node_modules/strapi-admin/admin/src/components/LeftMenu/LeftMenuHeader` folder and create a new admin folder `./admin/src/components/LeftMenu/LeftMenuHeader` and change the styles inside `./admin/src/components/LeftMenu/LeftMenuHeader/Wrapper.js`.

Reviewer: Maybe change it to something like this:

```suggestion
For example, to change the top-left displayed admin panel's color, copy the `./node_modules/strapi-admin/admin/src/components/LeftMenu/LeftMenuHeader` folder to `./admin/src/components/LeftMenu/LeftMenuHeader` (create these folders if they don't exist) and change the styles inside `./admin/src/components/LeftMenu/LeftMenuHeader/Wrapper.js`.
```

As this covers only your use case. What about people with an already existing `./admin/src/components` folder?
(You btw also removed the word 'to' which makes it an weird sounding sentence XD)
